packages/flutter/test/widgets/sliver_main_axis_group_test.dart

```diff
@@ -797,6 +797,64 @@ void main() {
     await tester.pumpAndSettle();
     expect(tester.getTopLeft(find.byKey(key)), Offset.zero);
   });
+
+  testWidgets('SliverMainAxisGroup scrolls to the correct position when focusing on a text field within a header', (WidgetTester tester) async {
```

Comment:
Can we add a test for GrowthDirection.reverse?

Reply:
For some reason it never goes into reverse. I tried to look at other examples, but there are no similar ones. And it's also a bit confusing on how it works.

Reply:
Can you try to reproduce the bug with GrowthDirection.reverse?

Reply:
I tried but I couldn't. I must probably be missing something or it never really goes into reverse.

Reply:
If it can't be reproduced in `GrowthDirection.reverse`, then we should probably not try to solve a problem that doesn't exist yet. In the meantime I'll try reproducing it.